cdap-ui/app/directives/my-post-run-action-wizard/my-post-run-action-wizard.js

```diff
@@ -31,24 +30,22 @@ angular.module(PKG.name + '.commons')
     };
   })
   .service('myPostRunActionWizardService', function($uibModal) {
-    this.show = (actionCreator, store, isEdit, action, isDisabled) => {
+    this.show = (actionCreator, store, mode, action) => {
       $uibModal.open({
         templateUrl: 'my-post-run-action-wizard/my-post-run-action-wizard-modal.html',
         backdrop: true,
         resolve: {
           rActionCreator: () => actionCreator || {},
-          rIsEdit: () => isEdit || false,
           rStore: () => store,
-          rIsDisabled: () => isDisabled || false,
-          rAction: () => action || null
+          rAction: () => action || null,
+          rMode: () => mode
         },
         size: 'lg',
         windowClass: 'post-run-actions-modal cdap-modal',
-        controller: ['$scope', 'rActionCreator', 'rIsEdit', 'rStore', 'rAction', 'rIsDisabled', function($scope, rActionCreator, rIsEdit, rStore, rAction, rIsDisabled) {
+        controller: ['$scope', 'rActionCreator', 'rStore', 'rMode', 'rAction', function($scope, rActionCreator, rStore, rMode, rAction) {
```

Comment:
Talked to @elfenheart. ngAnnotate cannot add angular annotations to anonymous functions. Moving this change to another PR as we have more than few places where we need to fix this to be consistent.